pkg/cmd/server/server.go

```diff
@@ -124,6 +131,10 @@ func NewCommand() *cobra.Command {
 	command.Flags().Var(logLevelFlag, "log-level", fmt.Sprintf("the level at which to log. Valid values are %s.", strings.Join(logLevelFlag.AllowedValues(), ", ")))
 	command.Flags().StringVar(&pluginDir, "plugin-dir", pluginDir, "directory containing Ark plugins")
 	command.Flags().StringVar(&metricsAddress, "metrics-address", metricsAddress, "the address to expose prometheus metrics")
+	command.Flags().Duration("backup-sync-period", config.backupSyncPeriod, "how often to ensure all Ark backups in object storage exist as Backup API objects in the cluster")
+	command.Flags().Duration("pod-vol-op-timeout", config.podVolumeOperationTimeout, "how long backups/restores of pod volumes should be allowed to run before timing out")
```

Comment:
don't love the flag name but I get that it's super-long if you spell it all out.  I wonder if we should just call this `restic-timeout`.  Probably way clearer.